README.md

```diff
@@ -160,12 +174,11 @@ mremi_contact:
         name:              contact_form
         validation_groups: [Default]
         subject_provider:  mremi_contact.subject_provider.noop
-        captcha_disabled:  false
-        captcha_type:      genemu_captcha # or genemu_recaptcha
+        captcha_type:      false
```

Comment:
You want me to remove captcha_type mentions in default configuration snippet and previous snippet entirely?